src/content/docs/apm/apm-ui-pages/monitoring/find-performance-problems-apm-transactions-page.mdx

```diff
@@ -0,0 +1,374 @@
+---
+title: 'Find performance problems using the APM Transactions page'
+tags:
+  - APM
+  - APM UI pages
+  - Monitoring
+translate:
+  - jp
+metaDescription: APM's Transactions page shows app, browser, and other transaction details, average response time, and throughput.
+redirects:
+  - /docs/applications-menu/web-transactions
+  - /docs/applications-menu/background-tasks
+  - /docs/applications-menu/transactions-dashboard
+  - /docs/apm/applications-menu/monitoring/transactions-dashboard
+  - /docs/apm/applications-menu/monitoring/transactions-page
+  - /docs/apm/applications-menu/monitoring/transactions-page-find-performance-problems
+  - /docs/apm/applications-menu/monitoring/transactions-page-find-specific-performance-problems
+  - /docs/apm/transactions/intro-transactions/transactions-page-find-specific-performance-problems
+  - /docs/apm/apm-ui-pages/monitoring/transactions-page-find-specific-performance-problems
+---
+
+The APM **Transactions** page helps you identify transactions to look at more closely in order to troubleshoot performance problems and avoid errors. 
+
+In general, the longer a given transaction takes, and the more time it takes compared to other transactions, the more likely it is that it's a source of issues for a given app. Similarly, low throughput can be an indicator that not everything is working as it should. 
+
+We recommend you make a habit of reviewing app performance data on the APM **Transactions** page more closely in order to scan for new issues and diagnose the cause of problems you're already aware of.
+
+<Callout variant="tip">
+  At New Relic, a transaction is defined as one logical unit of work in a software application. For more information about transactions in the context of APM, see [Transactions in New Relic's APM](/docs/apm/transactions/intro-transactions/transactions-new-relic-apm/). 
```

Comment:
I think this is probably a standard definition that Language folks have already reviewed, but I think some synonyms for Transaction are "endpoint" or "code path". Might be worth including those somewhere.

Reply:
We have a very complicated set of definitions here that another of our writers worked with a bunch of SMEs to lay out, so the place to address that would be by submitting a PR or issue proposing changes to /docs/apm/transactions/intro-transactions/transactions-new-relic-apm/ directly FYI. :)

Reply:
@rkbodenner ^^ just in case you want to pursue in a new PR or issue.